llrt_modules/src/modules/timers.rs

```diff
@@ -130,16 +130,19 @@ fn get_timer_state<'a>(
     unsafe { rt_timers.unwrap_unchecked() }
 }
 
-fn clear_timeout_interval(ctx: Ctx<'_>, id: usize) -> Result<()> {
```

Comment:
You can use the `Coerced` wrapper for that

Reply:
Hum, would that would return a NaN and go through the lookup unnecessary no?

Reply:
I guess it might